code/modules/vehicles/cars/clowncar.dm

```diff
@@ -53,16 +53,20 @@
 		var/mob/living/carbon/forced_mob = M
 		if(forced_mob.has_reagent(/datum/reagent/consumable/ethanol/irishcarbomb))
 			var/reagent_amount = forced_mob.reagents.get_reagent_amount(/datum/reagent/consumable/ethanol/irishcarbomb)
+			forced_mob.reagents.del_reagent(/datum/reagent/consumable/ethanol/irishcarbomb)
 			var/obj/item/organ/stomach/stomach = forced_mob.getorganslot(ORGAN_SLOT_STOMACH)
 			if(stomach)
 				reagent_amount += stomach.reagents.get_reagent_amount(/datum/reagent/consumable/ethanol/irishcarbomb)
-			if(reagent_amount >= 10)
-				forced_mob.reagents.del_reagent(/datum/reagent/consumable/ethanol/irishcarbomb)
 				stomach.reagents.del_reagent(/datum/reagent/consumable/ethanol/irishcarbomb)
-				message_admins("[ADMIN_LOOKUPFLW(forced_mob)] was forced into a clown car after drinking Irish Car Bomb, causing an explosion.")
-				forced_mob.log_message("was forced into a clown car after drinking Irish Car Bomb, causing an explosion.", LOG_GAME)
+			if(reagent_amount >= 10)
+				message_admins("[ADMIN_LOOKUPFLW(forced_mob)] was forced into a clown car with [reagent_amount] unit(s) of Irish Car Bomb, causing an explosion.")
+				forced_mob.log_message("was forced into a clown car with [reagent_amount] unit(s) of Irish Car Bomb, causing an explosion.", LOG_GAME)
 				audible_message(span_userdanger("You hear a rattling sound coming from the engine. That can't be good..."), null, 1)
```

Comment:
I'd recommend making this appear in runechat (so it's apparent to everyone inside) and lengthening the delay before the explosion (so that the clown and the other occupants will have time to ditch the car)

Reply:
to be fair, userdanger is pretty huge